GOFevaluation/evaluator_base.py

```diff
@@ -280,24 +288,28 @@ def _get_pvalue(self, n_perm=1000, d_min=None):
             warnings.warn(f'p-value is 0.0. (Observed GoF: '
                           f'{self.gof:.2e}, maximum of simulated GoFs: '
                           f'{max(fake_gofs):.2e}). For a more '
-                          f'precise result, increase n_mc!', stacklevel=2)
+                          f'precise result, increase n_perm!', stacklevel=2)
         elif pvalue == 1:
             warnings.warn(f'p-value is 1.0. (Observed GoF '
                           f'{self.gof:.2e}, minimum of simulated GoFs: '
                           f'{min(fake_gofs):.2e}). For a more '
-                          f'precise result, increase n_mc!', stacklevel=2)
+                          f'precise result, increase n_perm!', stacklevel=2)
```

Comment:
just a suggestion/comment on `n_mc` and `n_perm` definitions. The `get_pvalue` function, which turns the p-value of the gof evaluation, differs in the input variables depending on the evaluator_base considered. In particular if the model is expressed as a [pdf](https://github.com/XENONnT/GOFevaluation/blob/7822f1eb93e80fc3bae8b3cb0efdcf5743e8f043/GOFevaluation/evaluator_base.py#L221) the [`get_pvalue`](https://github.com/XENONnT/GOFevaluation/blob/7822f1eb93e80fc3bae8b3cb0efdcf5743e8f043/GOFevaluation/evaluator_base.py#L181) function takes as input `n_mc`, while if the model is a [sample](https://github.com/XENONnT/GOFevaluation/blob/7822f1eb93e80fc3bae8b3cb0efdcf5743e8f043/GOFevaluation/evaluator_base.py#L236) data-set, the [`get_pvalue`](https://github.com/XENONnT/GOFevaluation/blob/7822f1eb93e80fc3bae8b3cb0efdcf5743e8f043/GOFevaluation/evaluator_base.py#L301) takes `n_perm`. Even if the two operations differ in the method used to extract the p-value, do you think it is possible to standardize the name of this variable somehow?